modules/Material/Action.qml

```diff
@@ -69,6 +69,11 @@ QtObject {
     property bool visible: true
 
     /*!
+       Set to \c true to rotate the icon on mouse hover.
```

Comment:
Please add the number of degrees and the direction in which the rotation will happen to help make the documentation a bit more clear.